test/e2e/common.go

```diff
@@ -655,7 +658,9 @@ func runCommand(logFolder, filename, machineIP, user, command string) error {
 	if err != nil {
 		return fmt.Errorf("couldn't open a new session: %w", err)
 	}
-	logFile := path.Join(logFolder, filename)
+	if err := session.Run("sudo " + command + "\n"); err != nil {
+		return fmt.Errorf("unable to send command %q: %w", "sudo "+command, err)
+	}
```

Comment:
I may be missing something but isn't this doing the exact same thing as lines 668-670? Do we need both?